be/src/storage/update_manager.h

```diff
@@ -101,6 +101,10 @@ class UpdateManager {
 
     Status set_cached_delta_column_group(KVStore* meta, const TabletSegmentId& tsid, const DeltaColumnGroupPtr& dcg);
 
+    Status set_cached_empty_delta_column_group(KVStore* meta, const TabletSegmentId& tsid);
+
+    bool TEST_get_cached_delta_column_group(const TabletSegmentId& tsid, int64_t version, DeltaColumnGroupList* dcgs);
```

Comment:
`get_cached_delta_column_group` is a better name. Not need to add `TEST_` prefix.

Reply:
Done